src/main/scala/tests/packageSymbolSignatures.scala

```diff
@@ -0,0 +1,19 @@
+package tests
+
+package packageSymbolSignatures
```

Comment:
IIUC we see one pseudo-package-object for every file that has top-level definitions and afterwards we merge them, correct? Can we add a second file with top-level definitions to `tests` so that we can check the `DPackage` objects are merged correctly?

Reply:
Yes, that's correct. I'm gonna make some tests for it aswell.